### DISCUSSION TITLE
Running a server under path prefix?

### DISCUSSION THREAD

**Discussion by KristijanZic** [serverpod/serverpod#2462] [Q&A] [answered]
I'm trying to get the Serverpod api, insights and web running under a domain with a subdomain and a path prefix but I get 404 error. I'm guessing server expects to be ran from the root path "/" instead of "/v1/staging".

So the urls are `api.project.example.com/v1/staging`, `insights.project.example.com/v1/staging`, `web.project.example.com/v1/staging`.

I'm deploying to docker and this is the label I'm using:
```
- "traefik.http.routers.api-staging.rule=(Host(`api.project.example.com`) && PathPrefix(`/v1/staging/`))"
...
- "traefik.http.routers.api-staging.rule=(Host(`insights.project.example.com`) && PathPrefix(`/v1/staging/`))"
...
- "traefik.http.routers.api-staging.rule=(Host(`web.project.example.com`) && PathPrefix(`/v1/staging/`))"
```

How do I make it work with the path prefix? (it works fine without path prefix).

**Comment by SandPod** [comment, marked as answer]
Hey @KristijanZic!

Currently, we don't support running the server under a path prefix.
The server's routing relies on paths that are based on the root.

Please file an issue with a description of how you would like this to work if this would be a desired feature for the future.
